README.md

```diff
@@ -49,9 +49,10 @@ The possibility to automatically prepare such reports enabled us to notify entit
 ## Development
 To start a locally modified version of Artemis, run:
 ```commandline
- cp env.example .env  # after doing that, configure the settings by changing the user-agent and any other settings you want to change
  ./scripts/start_dev
 ```
+This script automatically copies the example environment file (.env.example) to .env if it doesn't exist. You can then configure the settings in the .env file according to your needs, including customizing the user-agent and other relevant parameters.
```

Comment:
.env.example -> env.example

provide how to customize user-agent ("by setting the CUSTOM_USER_AGENT variable") and info where are all the config vars: https://artemis-scanner.readthedocs.io/en/latest/user-guide/configuration.html